package.json

```diff
@@ -82,6 +84,8 @@
     "build": "tsc",
     "validate": "npm ls",
     "prepare": "npm run build",
-    "version": "manual-git-changelog onversion"
+    "version": "manual-git-changelog onversion",
+    "download-performance-data": "curl https://json-ld.org/test-suite/tests/toRdf-manifest.jsonld > perf/toRdf-manifest.jsonld && curl https://raw.githubusercontent.com/comunica/comunica/master/packages/actor-init-sparql/config/sets/sparql-init.json > perf/sparql-init.json",
```

Comment:
I think there's no need to have these perf tests in this repo. We can keep it at https://github.com/rubensworks/jsonld-streaming-parser.js/, run the perf tests there using this new validator lib, and report the perf results in the PR here.